_format/1.1/index.md

```diff
@@ -1970,6 +1969,19 @@ server, and we are defining its semantics for JSON:API.
 
 #### <a href="#crud-updating-relationship-responses" id="crud-updating-relationship-responses" class="headerlink"></a> Responses
 
+##### <a href="#crud-updating-relationship-responses-200" id="crud-updating-relationship-responses-200" class="headerlink"></a> 200 OK
+
+If a server accepts an update but also changes the targeted relationship(s)
```

Comment:
> Relationships in plural is confusing to me.

I used `relationship(s)` here because a single request can impact multiple one-to-many relationships between unique resources. Which I think aligns with what you're getting at here:

> Maybe this is not talking about a relationship but instead of the related resources identified by their resource identifier objects?

Reply:
After review, I'm changing from `relationship(s)` to `relationship`, since a request can in fact only affect one relationship but potentially multiple relationship members. Thanks for calling this out, @jelhan.